tests/test_accumulate.py

```diff
@@ -72,6 +72,7 @@ def accumulate_data(tmpdir_factory):
         ),
         dump_buffer,
         accumulate_params,
+        gdb=True,
```

Comment:
Is `test_accumulate` the only place that kotekan fails with `SIGILL`?  (without this `gdb` "hack"?)

Reply:
Correct. It's the only place where I've seen it, at least.